src/Model/Command/UserMerge.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\Model\Command;
+
+/**
+ * Merge two users. DELETES source user!
+ */
+class UserMerge extends AbstractCommand
+{
+    /** @var string */
+    private $sourceUserId;
+    /** @var string */
+    private $targetUserId;
+
+    /**
+     * Merge source user to target user AND DELETE SOURCE USER.
```

Comment:
Same as in #27. I think we should merge constructor and class comments in this particular case.

Reply:
So you'd like to have the PHPDoc as class comment or as constructor comment?

IMHO we may have some more detailed information and documentation in the class, and basic information on the constructor.